python/docs/source/user_guide/pandas_on_spark/supported_pandas_api.rst

```diff
@@ -362,7 +362,9 @@ Supported DataFrame APIs
 +--------------------------------------------+-------------+--------------------------------------+
 | :func:`replace`                            | P           | ``regex``, ``method``                |
 +--------------------------------------------+-------------+--------------------------------------+
-| resample                                   | N           |                                      |
+| resample                                   | P           |``axis``, ``convention``, ``kind``    |
```

Comment:
Can we add the link??

```
:func:`resample`
```

should work

Reply:
good catch! will update